src/manifest.json

```diff
@@ -2,7 +2,7 @@
     "manifest_version": 2,
 
     "name": "Tab Manager Shortcuts",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
Don't change the version number as maintainers should decide how to this is incremented.